### DISCUSSION TITLE
Getting label values in Render Note

### DISCUSSION THREAD

**Discussion by tevenfeng** [TriliumNext/Notes#352] [Q&A / Support] [answered]
Is it possible to get a lable value in render note? Does anyone know?

**Comment by meichthys** [comment]
Are you trying to do this using the ETAPI? or just via the GUI?
Via the gui you can view labels by selecting the Owned or Inherited Attributes Ribbon item:
![image](https://github.com/user-attachments/assets/69f0e702-4223-4648-9df7-29edcde729b1)

**Reply by tevenfeng** [reply]
Nope, maybe I'll just give you an example about my question.
I created a render note and set a relation to an HTML code note including the following codes:
```
<div style="position: relative; padding: 30% 45%;">
    <iframe style="position: absolute; width: 100%; height: 100%; left: 0; top: 0;"
            src="//player.bilibili.com/player.html?isOutside=true&bvid=<%note.getLabelValue('bvid')%>&p=1&poster=1&autoplay=0"
            scrolling="no"
            border="0"
            frameborder="no"
            framespacing="0"
            allowfullscreen="true"
            height="600px"
            width="800px">
    </iframe>
</div>
```
But somehow the code `bvid=<%note.getLabelValue('bvid')%>` doesn't work at all, bvid will never get the actual label value.

**Reply by zerebos** [reply, marked as answer]
The html for the render note is not treated as a template thus it doesn't receive anything like that. You can adjust that iframe url using the attribute by using an additional JS note attached as a child to the render note

**Reply by tevenfeng** [reply]
Thx a lot, finallly I've got it working
